### DISCUSSION TITLE
Use of margin factor for Cat 9 Downstream transportation and distribution emission

### DISCUSSION THREAD

**Discussion by arby1005** [USEPA/supply-chain-factors#53] [Q&A]
Hi all,

I am wondering if we can use the margin factor to estimate Cat 9 Downstream transportation and distribution emission (GHG Protocol). 

I am calculating Scope 3 emission for a manufacturer and they have no insight into the third-party outbound transportation which is paid by their customer, as well as the retailer emission. 

As the margin factor is the gate-to-shelf emission, it should cover the missing part. If it is possible, I wonder if we could use the selling price of the manufacturer's products to estimate the emissions?

Thank you!

**Comment by bl-young** [comment]
Yes that seems reasonable to me.
